examples/va/hci/edpm-pre-ceph/values.yaml

```diff
@@ -45,6 +45,10 @@ data:
           {%- endfor %}
           {% set min_viable_mtu = mtu_list | max %}
           network_config:
+          - type: interface
```

Comment:
we'll need to amend the ci-framework template in order to make the deploy aware of `nic1`:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/ci_gen_kustomize_values/templates/hci/edpm-values/values.yaml.j2#L28-L31

Reply:
https://github.com/openstack-k8s-operators/ci-framework/pull/1390 done here